Chart.yaml

```diff
@@ -14,11 +14,11 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 1.3.4
+version: 1.5.2
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 2.0.0
+appVersion: 2.2.0
```

Comment:
Did we decide to jump from 1.5.0 to 2.2.0?  Bumping it up to go with the HeLx version?

Reply:
The appVersion is supposed to correspond with an image tag.  In this case it should be 1.5.1.